include/pokemon.h

```diff
@@ -359,7 +359,7 @@ struct SpeciesInfo /*0x8C*/
  /* 0x05 */ u8 baseSpDefense;
  /* 0x06 */ u8 types[2];
  /* 0x08 */ u8 catchRate;
- /* 0x09 */ u8 padding1;
+ /* 0x09 */ u8 forceTeraType;
```

Comment:
Wouldn't this cause for anything that's set as 0 default to Normal type?

Reply:
From testing it seems every pokemon gets a tera type as normal if its not defined in their species.

Reply:
Oh I see what you mean now